client/landing/jetpack-cloud/sections/settings/main.jsx

```diff
@@ -3,22 +3,78 @@
  */
 import React, { Component } from 'react';
 import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
 import { getSelectedSiteId } from 'state/ui/selectors';
+import RewindCredentialsForm from 'components/rewind-credentials-form';
+import { Card } from '@automattic/components';
+import getRewindState from 'state/selectors/get-rewind-state';
+import QueryRewindState from 'components/data/query-rewind-state';
+import Gridicon from 'components/gridicon';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
 
 class SettingsPage extends Component {
 	render() {
-		return <div>Welcome to the settings page for { this.props.siteId }</div>;
+		const { rewind, siteId, translate } = this.props;
+
+		const connected = 'active' === rewind.state;
```

Comment:
Can we update this to `isConnected` ?

Reply:
done in https://github.com/Automattic/wp-calypso/pull/40094/commits/7f711af2e0c7a17edd40ceed07639db6fd914bb8